packages/terra-clinical-label-value-view/src/LabelValueView.jsx

```diff
@@ -21,6 +21,10 @@ const propTypes = {
   * If it is not valued, it will take false as a default, meaning that is is not a child of a description list.
   */
   isChildOfDescriptionList: PropTypes.bool,
+  /**
+  * The id of the element to be used as the accessible label for LabelValueView.
+  */
+  ariaLabelledby: PropTypes.string,
```

Comment:
small nit, can we capitalize By since it is technically another word for consistency with other components: https://github.com/cerner/terra-core/blob/main/packages/terra-toolbar/src/Toolbar.jsx#L32 
```suggestion
  ariaLabelledBy: PropTypes.string,
```